Makefile

```diff
@@ -156,9 +156,38 @@ kind-load-bundles: ## Load the e2e testdata container images into a kind cluster
 	${KIND} load docker-image testdata/bundles/plain-v0:invalid-crds-and-crs --name $(KIND_CLUSTER_NAME)
 	${KIND} load docker-image testdata/bundles/plain-v0:subdir --name $(KIND_CLUSTER_NAME)
 
-kind-load: ## Load-image loads the currently constructed image onto the cluster
+kind-load: ## Loads the currently constructed image onto the cluster
 	${KIND} load docker-image $(IMAGE) --name $(KIND_CLUSTER_NAME)
 
+##################################
+# CRDValidator Webhook Targets #
+##################################
+.PHONY: crdvalidator build-crdvalidator-image kind-load-crdvalidator install-crdvalidator deploy-crdvalidator
+
+##@ crdvalidator:
+
+crdvalidator: 
+	CGO_ENABLED=0 go build $(VERSION_FLAGS) -o $(BIN_DIR)/$@ ./cmd/crdvalidator
+
+CRD_WEBHOOK_IMAGE_REPO=quay.io/operator-framework/crd-validation-webhook
+CRD_WEBHOOK_IMAGE=$(CRD_WEBHOOK_IMAGE_REPO):main
+build-crdvalidator-image: ## Builds crd-validation-webhook container image locally
+	docker build -f Dockerfile.crdvalidator -t $(CRD_WEBHOOK_IMAGE) .
+
+kind-load-crdvalidator: ## Loads the currently constructed crd-validation-webhook onto the cluster
+	${KIND} load docker-image $(CRD_WEBHOOK_IMAGE) --name $(KIND_CLUSTER_NAME)
+
+install-crdvalidator: build-crdvalidator-image kind-load-crdvalidator ## Install the crdvalidator manifests to the current cluster. Requires that cert-manager is installed prior.
+	kubectl apply -f cmd/crdvalidator/manifests
+
+deploy-crdvalidator: cert-mgr install-crdvalidator ## Install the crdvalidator manifests to the current cluster and ensures that cert-manager is installed prior.
+
+test-crdvalidator-e2e: ginkgo ## Run the crdvalidator e2e tests. Assumes you have already installed the crdvalidator webhook.
+	$(GINKGO) -trace -progress $(FOCUS) cmd/crdvalidator/test/e2e
+
+crdvalidator-e2e: KIND_CLUSTER_NAME=crdvalidator-e2e
```

Comment:
This seems problematic at a first glance, but it's possible I'm reading this wrong: I don't think we create this kind cluster anywhere in the target prerequisite chain. I also don't think we need to separate out kind clusters when running the plain and crdvalidator e2e suites, at least for right now.